manifests/integrations/http_check.pp

```diff
@@ -185,6 +185,8 @@
   $check_certificate_expiration = true,
   $days_warning = undef,
   $days_critical = undef,
+  $check_hostname = undef,
```

Comment:
Please use typed arguments. Although we don't have types everywhere  yet, it's preferable to add them for new arguments.

Reply:
Done 👍